shared/common-adapters/button.desktop.js

```diff
@@ -136,6 +164,7 @@ const buttonCommon1 = {
 const buttonCommon2 = {
   ...globalStyles.DZ2.fontSemibold,
   color: globalColorsDZ2.white,
+  whiteSpace: 'nowrap',
```

Comment:
let's disallow our buttons wrapping and being all messed up. you can see this on master if you squish the view